problemsets-mat223/modules/module6-exercises.tex

```diff
@@ -16,6 +16,37 @@
 			\item $\mathcal Q \subseteq \R^n$, where $\mathcal Q$ is the complete solution to
 			$a_1x_1+a_2x_2+...+a_{n-1}x_{n-1}+a_nx_n=0$ where $a_1,...,a_n \in \R$.
 		\end{enumerate}
+		\begin{solution}
+			\begin{enumerate}
+				\item $\ST$ is a subspace: it is non-empty since $\mat{0\\0}\in\ST$. 
+				\begin{enumerate}
+					\item Let $\mat{x_1\\y_1},\mat{x_2\\y_2}\in \ST$. Then $3x_1-y_1=0=3x_2-y_2$. Hence $3(x_1+x_2)-(y_1+y_2)=(3x_1-y_1)+(3x_2-y_1)=0+0=0$ and $\mat{x_1\\y_1}+\mat{x_2\\y_2}\in\ST$.
+					\item Let $\mat{x\\y}\in\ST$. Then $3x-y=0$. Hence for all $\alpha\in\SR$, $3\alpha x-\alpha y=\alpha(3x-y)=\alpha\cdot 0=0$ and $\alpha\mat{x\\y}\in\ST$.
+				\end{enumerate}
+
+				\item $\SU$ is a subspace: it is non-empty since $\mat{0\\0}\in\SU$. 
+				\begin{enumerate}
+					\item Let $\mat{x_1\\y_1},\mat{x_2\\y_2}\in \SU$. Then $\frac{1}{2}x_1-6y_1=0=\frac{1}{2}x_2-6y_2$. Hence $\frac{1}{2}(x_1+x_2)-6(y_1+y_2)=(\frac{1}{2}x_1-6y_1)+(\frac{1}{2}x_2-6y_2)=0+0=0$ and $\mat{x_1\\y_1}+\mat{x_2\\y_2}\in\SU$.
```

Comment:
The proof should be written out in words; don't use an `enumerate` to shorten things. It is very important that we give very correct and very consistent examples of proofs here. See the module body for the format of every subspace proof.